### PR TITLE
chore: reshape packages folder hierarchy

### PR DESCRIPTION
**Pull request by jsomsanith-tlnd**
**What is the problem this PR is trying to solve?**
The packages folder is a mess --> try to reshape it

**What is the chosen solution to this problem?**
```
packages/
    cmf/ --> cmf related libs
    deprecated/ --> deprecated packages
    forms/ --> every lib related to forms (including jsfc)
    tools/ --> eslint config and generator (which is not in lerna anymore)
    webpack-plugins/ --> the webpack plugins :D 
```

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by vbocquet** [comment]
Agree,
Just a idea: maybe `containers` folder can move to `cmf`? It's CMF related, and not obvious at first.

File: package.json [hunk 1]
```diff
@@ -27,26 +27,26 @@
...
     "extract-i18n-components": "i18next-scanner --config packages/components/i18next-scanner.config.js 'packages/components/src/**/*.js'",
-    "extract-i18n-forms": "i18next-scanner --config packages/forms/i18next-scanner.config.js 'packages/forms/src/**/*.js'",
+    "extract-i18n-forms": "i18next-scanner --config packages/forms/i18next-scanner.config.js 'packages/forms/react-forms/src/**/*.js'",
     "extract-i18n-containers": "i18next-scanner --config packages/containers/i18next-scanner.config.js 'packages/containers/src/**/*.js'",
     "extract-i18n-datagrid": "i18next-scanner --config packages/datagrid/i18next-scanner.config.js 'packages/datagrid/src/**/*.js'",
-    "audit-router": "cd packages/router && yarn audit --json && cd ../../",
-    "audit-scripts": "cd packages/scripts && yarn audit -s --json && cd ../../",
-    "audit-html": "cd packages/html-webpack-plugin && yarn audit --json && cd ../../"
+    "audit-router": "cd packages/cmf/router && yarn audit --json && cd ../../",
+    "audit-scripts": "cd packages/deprecated/scripts && yarn audit -s --json && cd ../../",
```
Comment (jmfrancois) [inline package.json]: please drop this one; it is depreacted now

Comment (jsomsanith-tlnd) [reply package.json]: Done, thanks

**Comment by jsomsanith-tlnd** [comment]
This is too old, and too many conflicts. Let's reshape the folders once we have have the design system repo

**Pull request closed without merging**